internal/kafka/test/integration/kafka/kafka_test.go

```diff
@@ -22,6 +22,7 @@ func init() {
 }
 
 func TestMain(m *testing.M) {
+	exitCode := 0
```

Comment:
I made a similar change for `kas` as it has the same problem (but it does not manifest itself).